apps/profiles/migrations/0008_userrating.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.2.3 on 2023-12-16 09:35
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('profiles', '0007_user_is_email_verified'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='UserRating',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True, verbose_name='Created at')),
+                ('updated_at', models.DateTimeField(auto_now=True, null=True, verbose_name='Updated at')),
+                ('value', models.SmallIntegerField(default=0, null=True)),
+                ('comment', models.CharField(max_length=256, null=True)),
+                ('created_by', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='%(class)s_created', to=settings.AUTH_USER_MODEL, verbose_name='Created by')),
+                ('updated_by', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='%(class)s_updated', to=settings.AUTH_USER_MODEL, verbose_name='Updated by')),
+                ('user_rated', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='user_rated', to=settings.AUTH_USER_MODEL)),
+                ('user_rater', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='user_rater', to=settings.AUTH_USER_MODEL)),
```

Comment:
updated_by - looks redundant

created_by and user_rater look like the same, don't they?

Reply:
Yes, completely